set.mm

```diff
@@ -496112,125 +496221,216 @@ such that every prime ideal contains a prime element (this is a
+      irngnzply1lem.x $e |- ( ph -> X e. B ) $.
+      $( In the case of a field ` E ` , a root ` X ` of some non-zero
+         polynomials ` P ` with coefficients in a subfield ` F ` is integral
+         over ` F ` .  (Contributed by Thierry Arnoux, 5-Feb-2025.) $)
+      irngnzply1lem $p |- ( ph -> X e. ( E IntgRing F ) ) $=
+        ( cfv vp cirng co wcel wceq cress cmn1 wrex cdg1 cco1 cinvr cascl cmulr
+        cv cpl1 crg cuc1p csdrg cdr csubrg issdrg simp3bi syl drngringd cbs c0g
+        wne cdm cpws crh wf ccrg fldcrngd w3a simp2d eqid evls1rhm syl2anc rhmf
+        sylib fdmd eleqtrd ressply10g neeqtrd drnguc1p syl3anc uc1pmon1p fveq2d
+        simpr fveq1d eqeq1d csca casa cfield fldsdrgfld ply1assa assaring clmod
+        ply1lmod cn0 deg1nn0cl coe1fvalcl deg1ldg drnginvrcld ply1sca ffvelcdmd
+        asclf evls1muld oveq2d crngringd cvv fvexi a1i pwselbas ringrz rspcedvd
+        wa 3eqtrd elirng mpbir2and ) AGDEUBUCUDGBUDGUAUNZFTZTZHUEZUADEUFUCZUGTZ
+        UHSAYDGCYEUITZTZCUJTZTZYEUKTZTZYEUOTZULTZTZCYMUMTZUCZFTZTZHUEUAYQYFAYEU
+        PUDZCYEUQTZUDZYQYFUDAYEAEDURTUDZYEUSUDZNUUCDUSUDZEDUTTUDZUUDDEVAZVBVCZV
+        DZAUUDCYMVETZUDZCYMVFTZVGZUUBUUHACFVHUUJPAUUJDBVIUCZVETZFAFYMUUNVJUCUDZ
+        UUJUUOFVKADVLUDUUFUUPADMVMZAUUEUUFUUDAUUCUUEUUFUUDVNNUUGVTVOZBFEDUUNYEY
+        MJOUUNVPZYEVPZYMVPZVQVRUUJUUOYMUUNFUUJVPZUUOVPZVSVCZWAWBZACIUULQAUUJDDU
+        OTZEYMYEIUVFVPUUTUVAUVBUURKWCWDZUUJUUAYMYECUULUVAUVBUULVPZUUAVPZWEWFYNU
+        UAYGYMYEYPYKYFCUVIYFVPUVAYPVPZYNVPZYGVPZYKVPZWGVRAYAYQUEZXQZYCYSHUVOGYB
+        YRUVOYAYQFAUVNWIWHWJWKAYSGYOFTZTZGCFTTZDUMTZUCUVQHUVSUCZHAUUJGFEDUVSYPY
+        EBYOCYMJOUVAUUTUVBUVJUVSVPZUUQUURAYMWLTZVETZUUJYLYNAYNUUJUWBUWCYMUVKUWB
+        VPAYMWMUDZYMUPUDAYEVLUDUWDAYEADWNUDUUCYEWNUDZMNEDWOVRZVMYMYEUVAWPVCYMWQ
+        VCAYTYMWRUDUUIYMYEUVAWSVCUWCVPUVBXGAYLYEVETZUWCAUWGYEYKYJYEVFTZUWGVPZUW
+        HVPZUVMUUHAUUKYHWTUDZYJUWGUDUVEAYTUUKUUMUWKUUIUVEUVGUUJYGYMYECUULUVLUVA
+        UVHUVBXAWFYIUUJYMYECUWGYHYIVPZUVBUVAUWIXBVRAYTUUKUUMYJUWHVGUUIUVEUVGYIU
+        UJYGYMYECUWHUULUVLUVAUVHUVBUWJUWLXCWFXDAYEUWBVEAUWEYEUWBUEUWFYMYEWNUVAX
+        EVCWHWBXFZUVESXHAUVRHUVQUVSRXIADUPUDUVQBUDUVTHUEADUUQXJABBGUVPABDBUUOWN
+        UVPUUNXKUUSOUVCMBXKUDABDVEOXLXMAUUJUUOYOFUVDUWMXFXNSXFBDUVSUVQHOUWALXOV
+        RXRXPABDEYEUAFGHJUUTOLUUQUURXSXT $.
+    $}
+
+    $d .0. x $.  $d E p x $.  $d F p x $.  $d O p x $.  $d Z x $.  $d p ph x $.
+    $( In the case of a field ` E ` , the roots of non-zero polynomials ` p `
+       with coefficients in a subfield ` F ` are exactly the integral elements
+       over ` F ` .  Roots of nonzero polynomials are called algebraic numbers,
+       so this shows that in the case of a field, elements integral over ` F `
+       are exactly the algebraic numbers.  In this formula, ` dom O `
+       represents the polynomials, and ` Z ` the zero polynomial.  (Contributed
+       by Thierry Arnoux, 5-Feb-2025.) $)
+    irngnzply1 $p |- ( ph -> ( E IntgRing F )
```

Comment:
inconsistent non-zero and nonzero -> change to nonzero (same for lemma)

Reply:
Both fixed.